### PR TITLE
chore(ampd): include libssl3 in docker image

### PR DESCRIPTION
**Pull request by jcs47**
## Todos

- [ ] Unit tests
- [ ] Manual tests
- [ ] Documentation
- [ ] Connect epics/issues

### CODE REVIEW

File: ampd/Dockerfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ COPY ./ampd/build.rs ./ampd/build.rs
 RUN cargo install --locked --path ./ampd
 
 FROM debian:bookworm-slim AS runner
+RUN apt update && apt install libssl3
```
Comment (milapsheth) [inline ampd/Dockerfile:25]: why is this needed now? There's already an apt update and install at the top

Comment (milapsheth) [reply ampd/Dockerfile:25]: Looks like the previous apt install is for a different image

Comment (jcs47) [reply ampd/Dockerfile:25]: yes, currently we are unable to execute ampd in the runner image because libssl3 is missing.

**Review by milapsheth — APPROVED** [review]

File: ampd/Dockerfile (hunk 1 above)
Comment (milapsheth) [reply ampd/Dockerfile:25]: maybe because we don't pin the bookworm-slim version
